Assets/Plugins/Editor/Uplift/UpliftManager.cs

```diff
@@ -91,8 +91,9 @@ public struct DependencyState
 
 		public void InstallDependencies(InstallStrategy strategy = InstallStrategy.UPDATE_LOCKFILE)
 		{
-			PackageRepo[] targets = GetTargets(GetDependencySolver(), strategy);
-			InstallPackages(targets);
+			bool canUpdatePackages = (strategy != InstallStrategy.ONLY_LOCKFILE);
+			PackageRepo[] targets = GetTargets(GetDependencySolver(), strategy, canUpdatePackages);
```

Comment:
This changes `GetTargets` behavior so I'd rather not have this if this not related.